.github/workflows/test.yml

```diff
@@ -43,7 +47,17 @@ jobs:
         EOF
 
     - name: Build source rpm
+      id: srpm
       uses: ./
       with:
         tarball: action-build-srpm-1.tar.gz
         specfile: test.spec
+        sourcefiles: |
+          readme.md
+          readme2.md
```

Comment:
Can you put this to the example in the readme file? After that, feel free to merge it.